dashboard/test/ui/features/videoplayer.feature

```diff
@@ -19,9 +19,9 @@ Scenario: Flash fallback player gets injected in Chrome (assuming Flash is avail
 
 # no_ie because YouTube's recent change to their player is causing our
 #   fallback detection to get a false negative
-@no_ie
 # skip because SauceLabs is currently blocking YouTube content, causing
-# this to consistently fail. TODO @elijah: unskip when they update
+#   this to consistently fail. TODO @elijah: unskip when they update
+@no_ie
```

Comment:
Apparently not!
`features/videoplayer.feature: Parse error at features/videoplayer.feature:23. Found comment when expecting one of: examples, feature, scenario, scenario_outline, tag. (Current state: tag). (Cucumber::Core::Gherkin::ParseError)`